src/helpers/gho-launch/constants/GhoCCIPChains.sol

```diff
@@ -0,0 +1,230 @@
+        owner: GovernanceV3Arbitrum.EXECUTOR_LVL_1,
+        ccipRouter: CCIPChainRouters.ARBITRUM
+      });
+  }
+
+  /**
+   * @notice Returns the ChainInfo constant for Base
+   * @return ChainInfo The ChainInfo constant for Base
+   */
+  function BASE() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.BASE,
+        ghoToken: GhoBase.GHO_TOKEN,
+        ghoCCIPTokenPool: GhoBase.GHO_CCIP_TOKEN_POOL,
+        ghoBucketSteward: GhoBase.GHO_BUCKET_STEWARD,
+        ghoAaveCoreSteward: GhoBase.GHO_AAVE_CORE_STEWARD,
+        ghoCCIPSteward: GhoBase.GHO_CCIP_STEWARD,
+        aclManager: address(AaveV3Base.ACL_MANAGER),
+        tokenAdminRegistry: CCIPChainTokenAdminRegistries.BASE,
+        owner: GovernanceV3Base.EXECUTOR_LVL_1,
+        ccipRouter: CCIPChainRouters.BASE
+      });
+  }
+
+  /**
+   * @notice Returns the ChainInfo constant for Avalanche
+   * @return ChainInfo The ChainInfo constant for Avalanche
+   */
+  function AVALANCHE() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.AVALANCHE,
+        ghoToken: GhoAvalanche.GHO_TOKEN,
+        ghoCCIPTokenPool: GhoAvalanche.GHO_CCIP_TOKEN_POOL,
+        ghoBucketSteward: GhoAvalanche.GHO_BUCKET_STEWARD,
+        ghoAaveCoreSteward: GhoAvalanche.GHO_AAVE_CORE_STEWARD,
+        ghoCCIPSteward: GhoAvalanche.GHO_CCIP_STEWARD,
+        aclManager: address(AaveV3Avalanche.ACL_MANAGER),
+        tokenAdminRegistry: CCIPChainTokenAdminRegistries.AVALANCHE,
+        owner: GovernanceV3Avalanche.EXECUTOR_LVL_1,
+        ccipRouter: CCIPChainRouters.AVALANCHE
+      });
+  }
+
+  /**
+   * @notice Returns the ChainInfo constant for Gnosis
+   * @return ChainInfo The ChainInfo constant for Gnosis
+   */
+  function GNOSIS() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.GNOSIS,
+        ghoToken: GhoGnosis.GHO_TOKEN,
+        ghoCCIPTokenPool: GhoGnosis.GHO_CCIP_TOKEN_POOL,
+        ghoBucketSteward: GhoGnosis.GHO_BUCKET_STEWARD,
+        ghoAaveCoreSteward: GhoGnosis.GHO_AAVE_CORE_STEWARD,
+        ghoCCIPSteward: GhoGnosis.GHO_CCIP_STEWARD,
+        aclManager: address(AaveV3Gnosis.ACL_MANAGER),
+        tokenAdminRegistry: CCIPChainTokenAdminRegistries.GNOSIS,
+        owner: GovernanceV3Gnosis.EXECUTOR_LVL_1,
+        ccipRouter: CCIPChainRouters.GNOSIS
+      });
+  }
+
+  /**
+   * @notice Returns the ChainInfo constant for Ink
+   * @return ChainInfo The ChainInfo constant for Ink
+   */
+  function INK() public pure returns (ChainInfo memory) {
+    return
+      ChainInfo({
+        chainSelector: CCIPChainSelectors.INK,
+        ghoToken: 0xfc421aD3C883Bf9E7C4f42dE845C4e4405799e73,
+        ghoCCIPTokenPool: 0xDe6539018B095353A40753Dc54C91C68c9487D4E,
+        ghoBucketSteward: 0xA5Ba213867E175A182a5dd6A9193C6158738105A,
+        ghoAaveCoreSteward: address(0),
```

Comment:
provably makes sense, to leave a comment here on why address(0) is used

Reply:
Added a comment here https://github.com/bgd-labs/aave-proposals-v3/pull/802/commits/cad90fbe1fd0182041702e5d9b1788d4fcf34425